upload/modules/CraftingStore/src/Navigation/Builder/BackendNavigationBuilder.php

```diff
@@ -25,7 +25,7 @@ public function build(Navigation $navigation, User $user): void
                 if ($user->hasPermission(PermissionEnum::SETTINGS)) {
                     $order = $this->navigationOrderRetriever->retrieve();
 
-                    $navigation->add('craftingstore_divider', mb_strtoupper($this->craftingStoreLanguage->get(LanguageEnum::PREFIX, LanguageEnum::CRAFTING_STORE)), 'divider', 'top', null, $order, '');
+                    $navigation->add('craftingstore_divider', mb_strtoupper($this->craftingStoreLanguage->get(LanguageEnum::PREFIX, LanguageEnum::CRAFTING_STORE)), 'divider', 'top', null, $order);
```

Comment:
Why did you change this line?

Reply:
As PHPStorm suggests `Remove redundant arguments`